index.html

```diff
@@ -196,7 +202,7 @@ <h4>About me</h4>
           like what you see and have a project you need coded, don’t hesitate to
           contact me.
         </p>
-        <button type="button" class="cards-btn">Get My Resume</button>
+        <button type="button" class="cards-btn" >Get My Resume</button>
       </div>
       <hr class="divider" />
       <div class="technologies">
```

Comment:
- [ ] Nice work on the About Us section, kindly reduce the padding in between the `list` so it aligns with the Figma Design

## Your Design
![sk-1](https://user-images.githubusercontent.com/48371156/166103939-eea82f72-cd73-4baf-b68d-aa67a966ae19.png)

## Figma Design
![sk-2](https://user-images.githubusercontent.com/48371156/166104035-27992204-b2d8-4c58-8883-3b12c6404936.png)